moveit_core/robot_trajectory/include/moveit/robot_trajectory/robot_trajectory.h

```diff
@@ -324,7 +324,7 @@ class RobotTrajectory
       duration_iterator++;
       return *this;
     }
-    Iterator operator++(int)
+    Iterator (operator++)static_cast<int>
```

Comment:
I don't think this is a cast.  It is a function overload.

Reply:
woops.. my bad. Undid it.